<?xml version="1.0"?>

<launch>
	<include file="$(find experimental_2)/launch/rviz_gmapping.launch"></include>
	<include file="$(find experimental_2)/launch/move_base.launch"></include>
	<node name="markerDetector" pkg="experimental_2" type="markerDetector" respawn="false" output="screen"></node>
	<node name="findMarkerAction" pkg="experimental_2" type="findMarkerAction.py" respawn="false" output="screen"></node>
	<!--
	<include file="$(find experimental_2)/launch/plan.launch"></include>
	<node name="planLauncher" pkg="experimental_2" type="planLauncher" respawn="false" output="screen"></node>
	-->
</launch>
